docs/releasing.md

```diff
@@ -0,0 +1,80 @@
+# Release Process
+
+## Change milestone
+ -  Create a new GitHub milestone for the next release
+ - Change milestone applier so new changes can be applied to the appropriate release
+      - Open a PR in https://github.com/kubernetes/test-infra to change this [line](https://github.com/kubernetes/test-infra/blob/25db54eb9d52e08c16b3601726d8f154f8741025/config/prow/plugins.yaml#L344)
+        - Example PR: https://github.com/kubernetes/test-infra/pull/16827
+
+## Prepare branch, tag and release notes
+ - Identify a known good commit on the main branch
+ - Fast-forward the release branch to the selected commit. :warning: Always release from the release branch and not from master!
+   - `git checkout release-0.x`
+   - `git fetch upstream`
+   - `git merge --ff-only upstream/master`
+   - `git push`
+ - Create tag with git
+   - `export RELEASE_TAG=v0.4.6` (the tag of the release to be cut)
+   - `git tag -s ${RELEASE_TAG} -m "${RELEASE_TAG}"`
+	 - `-s` creates a signed tag, you must have a GPG key [added to your GitHub account](https://docs.github.com/en/enterprise/2.16/user/github/authenticating-to-github/generating-a-new-gpg-key)
+   - `git push upstream ${RELEASE_TAG}`
+ - Update the file `metadata.yaml` if is a major or minor release
```

Comment:
this we need to commit or the `make release` commits this change?

Reply:
Indeed, updated to reflect it needs to be committed and also updated to avoid the hard requirement for a release branch.

Reply:
thanks!